execution_workflows/LABRAT/modules/labrat_calculatepsi.nf

```diff
@@ -8,20 +8,26 @@ process LABRAT_CALCULATEPSI {
         publishDir "${params.outdir}/labrat", mode: params.publish_dir_mode
         container "quay.io/biocontainers/labrat:0.2.2--pyhdfd78af_0"
 
+        cpus 3
```

Comment:
same question here about hardcoding the numebr of cpus for this process as the question below :D

Reply:
Yeah I thought it made sense when executing some samples. 
Though as mentioned below, I can remove this statement :)